cpp/benchmarks/copying/shift.cu

```diff
@@ -68,6 +68,17 @@ static void BM_shift(benchmark::State& state)
     cuda_event_timer raii(state, true);
     auto output = cudf::shift(input, offset, *fill);
   }
+
+  auto const elems_read    = (size - offset);
+  auto const elems_written = use_validity ? (size - offset) : size;
```

Comment:
nit: Why is the `elems_written` changing based on `use_validity`?

Reply:
If `use_validity` is false, the fill value is a number, and the full column must be written. If `use_validity` is true, the filled values can be null, and writing only the elements which can be shifted + the bitmask would be sufficient.

Reply:
Can you add this comment into the code please?